tests/unit/test-class-wc-payments-account.php

```diff
@@ -906,7 +907,7 @@ public function test_handle_loan_approved_inbox_note_created_when_loan_summary_r
 				]
 			);
 
-		$notes_class = WC_Payment_Woo_Compat_Utils::get_notes_class();
+		$notes_class = Notes::class;
```

Comment:
Similar to notest classes here.

Reply:
Made changes here 
[9434390](https://github.com/Automattic/woocommerce-payments/pull/5331/commits/9434390dc3696f83f9b89ab8d94fdac0de98b8d3)